realm/realm-library/src/main/java/io/realm/internal/TableView.java

```diff
@@ -789,6 +789,13 @@ public Table pivot(long stringCol, long intCol, PivotType pivotType){
         return result;
    }
 
+    public TableView getDistinctView(long columnIndex) {
```

Comment:
Need Javadoc. All public methods should have javadoc even if they are in the internal API.